cyber/record/file/record_file_integration_test.cc

```diff
@@ -0,0 +1,126 @@
+/******************************************************************************
+ * Copyright 2018 The Apollo Authors. All Rights Reserved.
```

Comment:
2019.

Reply:
Sorry for the delay. The change is made.